src/vtcomposite.cpp

```diff
@@ -26,12 +29,14 @@ struct TileObject
     TileObject(std::uint32_t z0,
                std::uint32_t x0,
                std::uint32_t y0,
-               Napi::Buffer<char> const& buffer)
+               Napi::Buffer<char> const& buffer,
+               std::vector<std::string> layers0)
         : z{z0},
           x{x0},
           y{y0},
           data{buffer.Data(), buffer.Length()},
-          buffer_ref{Napi::Persistent(buffer)}
+          buffer_ref{Napi::Persistent(buffer)},
+          layers{std::move(layers0)}
```

Comment:
Why does this move the `layers0` argument instead of using a const reference?

Reply:
Good question! I'm open to ideas here - the clang tidy/format builds told me to use `std::move` here, so I'm admittedly just plugging that in. Can you show me how I'd use a const reference?

Reply:
What I was talking about was https://github.com/mapbox/vtcomposite/commit/05cbd11bd093d, but you are right and I am wrong, and the `TileObject` needs to contain its own copy of the vector instead of holding a reference back to the caller's vector, which apparently goes out of scope before it is used. So please ignore this comment.

Reply:
👍 thanks for the commit + trying it out @ericfischer!